.github/ISSUE_TEMPLATE/report.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Report
+about: To help identify and reproduce issues
+title: "[Issue]"
+labels: bug, bug-needs-triage, qa
+assignees: tommyv1987
+
+---
+
+**Describe the issue**
+A clear and concise description of what the issue is...
+
+**Expected behaviour**
+A clear and concise description of what you expected to happen...
+
+**Stack Traces**
+If there are stack traces or logs, please provide them here...
+
+**Steps to Reproduce**
+Steps to reproduce the behaviour, if you're familiar with BDD syntax, please write it in this style:
+- Given I was doing X
+- And I installed Y
+- When I actioned Y
+- Then I expect Z
+
+*An example:*
+- Given I was setting up a mix-node following the instructions in the docs
+- And I successfully bonded my node via the the wallet
+-  When I went to start my mixnode
+- Then I was presented with an error
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem...
+
+**Which area / component of Nym were you using?**
+ - [e.g. Gateway, Node, Client]
+ - OS: [e.g. Ubuntu 20.x]
+ - Version of binary [e.g. 0.11.0]
+
+**If this issue is a front end problem or any other area...**
```

Comment:
I'd perhaps avoid term "front end" as an average person might be unsure what that meant. Perhaps ask whether it's a UI / application problem?

Reply:
I will consolidate this section into one. Good suggestion.